docs/modules/ROOT/pages/proxies.adoc

```diff
@@ -97,7 +97,7 @@ This allows developers to add features, update logic, and fix bugs without touch
 
```

Comment:
Capitalize bullet list sentences suggestion.

Reply:
Also, in the quickstart there is a python code entry with an extra tabulation that is not needed:

<img width="818" alt="Screenshot 2022-09-29 at 23 17 05" src="https://user-images.githubusercontent.com/26070224/193143967-08fab086-8382-4161-8976-bbf1636c9817.png">

Reply:
We also have this indentation issue in **accounts.doc**

<img width="770" alt="Screenshot 2022-09-30 at 19 14 57" src="https://user-images.githubusercontent.com/26070224/193322899-6102b462-a3f2-4b1b-81cb-f6301b839871.png">